### ISSUE TITLE
solodus is not quoted.

### ISSUE THREAD

**Issue by ldn-softdev** [zaach/jsonlint#113]
Hi, a solidus char: '/' in JSON, as per:
http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf
is required to be escaped too, however your validation tool let it pass unescaped.

**Comment by c22** [comment]
No, solidus does not need to be escaped. As per the document you linked:

> the code points that must be escaped: quotation mark (U+0022), reverse solidus (U+005C), and the control characters U+0000 to U+001F

Solidus does not fall into this category. You CAN escape a solidus but it is not required.
